Makefile

```diff
@@ -74,6 +74,14 @@ lint: $(GOLANGCI_LINT)
 	$(GO_ENV_VARS) $(GOLANGCI_LINT) run
 	git --no-pager diff --exit-code
 
+.PHONY: test-go
+test: test-go
+test-go:
+	@ echo; echo "### $@:"
+	# TODO: After issue https://github.com/microsoft/go-winio/pull/169 is resolved, remove the filter on the test path.
```

Comment:
I was thinking we could have the target set as the following which is more inclusive at the same time worksarounds the issues with go-winio:

`@go test -race -count=1 $(shell go list ./... | grep -v integrationtests | grep -v server$$)`


just not including the integration tests and files at the top level of server directory (that seems to be causing the issue with winio) but includes everthing else and other directories in server directory…

Also the -race to check for races and -count to ensure that cache is not used ?

Reply:
it is still giving error when run go list 
"build golang.org/x/sys/windows: cannot load golang.org/x/sys/windows: no Go source files"